src/ansys/geometry/core/shapes/curves/circle.py

```diff
@@ -161,15 +161,32 @@ def transformed_copy(self, matrix: Matrix44) -> "Circle":
             New circle that is the transformed copy of the original circle.
         """
         new_point = self.origin.transform(matrix)
-        new_reference = self._reference.transform(matrix)
-        new_axis = self._axis.transform(matrix)
+
```

Comment:
Let's discuss this in a call... After thinking about it, it might be necessary to translate the axis and references too.